### PR TITLE
[Docs]:  Quick start docs and fix problem in docker/build.sh

### PR DESCRIPTION
**Pull request by baiyangtx**
## Why are the changes needed?

Add docs for quick start.

## Brief change log

- Add docs for the Quick start 
- Fix problem in build docker images in macos.

## How was this patch tested?

-  Run test locally before making a pull request

## Documentation

  - If yes, how is the feature documented? ( docs )

### CODE REVIEW

File: site/docs/ch/quickstart/cdc-ingestion.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# CDC Ingestion
+
+本部分文档将为您演示通过 Flink CDC 同步 MySQL 数据变更到 Arctic 的过程。本部分文档采用的是 
+[lakehouse-benchmark](https://github.com/NetEase/lakehouse-benchmark)项目中提供的 TPCC 数据集，
+模拟真实业务场景下对 MySQL 的读写， 并使用 
+[lakehouse-benchmark-ingestion](https://github.com/NetEase/lakehouse-benchmark-ingestion) 工具完成数据通过 Binlog 从 MySQL 到 Arctic 的同步。
+您需要提前通过 Setup from docker 完成集群的部署，并完成 Quick Demo 中，创建 Catalog 并开启 Optimizer 的部分。
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/cdc-ingestion.md:7]:
```suggestion
您需要提前通过 [Setup from docker](setup/setup-from-docker.md) 完成集群的部署，并完成 [Quick Demo](quick-demo.md) 中，创建 Catalog 并开启 Optimizer 的部分。
```

File: site/docs/ch/quickstart/cdc-ingestion.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+```shell
+
+docker exec -it lakehouse-benchmark java \
+  -jar lakehouse-benchmark.jar -b tpcc,chbenchmark \
+  -c config/mysql/sample_chbenchmark_config.xml \
+  --create=true --load=true
+  
+```
+
+等待命令执行完成，以上命令会在 MySQL 中初始化一个 oltpbench 的数据库，并且创建一系列业务表，并完成表数据的初始化
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/cdc-ingestion.md:24]:
```suggestion
等待命令执行完成，以上命令会在 MySQL 中初始化一个 oltpbench 的数据库，并且创建一系列业务表，并完成表数据的初始化。
```

File: site/docs/ch/quickstart/cdc-ingestion.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+# Step2. start streaming ingestion
+
+登录到 Ingestion 容器，通过以下命令完成 Ingestion 任务的启动
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/cdc-ingestion.md:28]:
```suggestion
登录到 Ingestion 容器，通过以下命令完成 Ingestion 任务的启动：
```

File: site/docs/ch/quickstart/cdc-ingestion.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+[lakehouse-benchmark](https://github.com/NetEase/lakehouse-benchmark)项目中提供的 TPCC 数据集，
+模拟真实业务场景下对 MySQL 的读写， 并使用 
+[lakehouse-benchmark-ingestion](https://github.com/NetEase/lakehouse-benchmark-ingestion) 工具完成数据通过 Binlog 从 MySQL 到 Arctic 的同步。
+您需要提前通过 Setup from docker 完成集群的部署，并完成 Quick Demo 中，创建 Catalog 并开启 Optimizer 的部分。
+
+
+# Step1. initialize tables
+
+
+通过以下命令完成测试数据的初始化。
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/cdc-ingestion.md:13]:
```suggestion
通过以下命令完成测试数据的初始化：
```

File: site/docs/ch/quickstart/cdc-ingestion.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+docker exec -it lakehouse-benchmark-ingestion bash
+
+java -jar lakehouse-benchmark-ingestion-1.0-SNAPSHOT.jar \
+  -confDir /usr/lib/lakehouse_benchmark_ingestion/conf \
+  -sinkType arctic \
+  -sinkDatabase oltpbench
+```
+
+开启后可以在 AMS Tables 页面查看到 Table 信息已经同步到 Arctic ，您可以通过 Terminal 执行 SQL 以查询 Arctic 上同步的数据。
+Ingestion 容器也是一个 Flink Job， 您可以通过 http://localhost:8082  访问 Flink Web UI 以查看 Ingestion 任务信息。
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/cdc-ingestion.md:40]:
```suggestion
Ingestion 容器也是一个 Flink Job， 您可以通过 [Flink Dashboard](http://localhost:8082) 访问 Flink Web UI 以查看 Ingestion 任务信息。
```

File: site/docs/ch/quickstart/cdc-ingestion.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+
+# Step3. start tpcc benchmark
+
+重新回到 Benchmark 容器，通过以下命令可以持续在测试库上执行有业务含义的 OLTP 操作
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/cdc-ingestion.md:45]:
```suggestion
重新回到 Benchmark 容器，通过以下命令可以持续在测试库上执行有业务含义的 OLTP 操作：
```

File: site/docs/ch/quickstart/cdc-ingestion.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+java -jar lakehouse-benchmark.jar -b tpcc,chbenchmark \
+  -c config/mysql/sample_chbenchmark_config.xml -\
+  -execute=true
+```
+
+此命令会一直不断的在测试库上执行 OLTP 操作，直到程序退出。
+此时可以回到 AMS 的 Terminal 页面，通过 Spark SQL 查询到  MySQL 上的数据变更会随着 Ingestion 任务不断的同步到 Arctic Table 上。
+
+注：Ingestion 任务的 Checkpoint 周期为 60s,  所以 Arctic 数据湖和 MySQL 的数据变更有 60s 的延迟
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/cdc-ingestion.md:58]:
```suggestion
???+note "Ingestion 任务的 Checkpoint 周期为 60s,  所以 Arctic 数据湖和 MySQL 的数据变更有 60s 的延迟。"
```

File: site/docs/ch/quickstart/cdc-ingestion.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+
+# Step 4.  check table result
+
+整个 tpcc benchmark 会执行 10min，在 tpcc benchmark 执行完成后，可以通过以下命令登录 mysql 容器
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/cdc-ingestion.md:63]:
```suggestion
整个 TPCC Benchmark 会执行 10min，在 TPCC Benchmark 执行完成后，可以通过以下命令登录 MySQL 容器：
```

File: site/docs/ch/quickstart/cdc-ingestion.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+  -execute=true
+```
+
+此命令会一直不断的在测试库上执行 OLTP 操作，直到程序退出。
+此时可以回到 AMS 的 Terminal 页面，通过 Spark SQL 查询到  MySQL 上的数据变更会随着 Ingestion 任务不断的同步到 Arctic Table 上。
+
+注：Ingestion 任务的 Checkpoint 周期为 60s,  所以 Arctic 数据湖和 MySQL 的数据变更有 60s 的延迟
+
+
+# Step 4.  check table result
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/cdc-ingestion.md:61]:
```suggestion
# Step 4. check table result
```

File: site/docs/ch/quickstart/quick-demo.md [hunk 10]
```diff
@@ -0,0 +1,169 @@
+# Setup
+
+在尝试使用 Arctic 之前，可以需要进行一些步骤来完成必要的环境准备，这里提供了两种方式来完成 Quick Start Demo 所必须的准备工作。
+
+1. [Setup from docker](./setup-from-docker.md)
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/quick-demo.md:5]:
```suggestion
1. [Setup from docker](setup/setup-from-docker.md)
```

File: site/docs/ch/quickstart/quick-demo.md [hunk 11]
```diff
@@ -0,0 +1,169 @@
...
+
+2. [Setup from binary release](./setup-from-binary-release.md)
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/quick-demo.md:7]:
```suggestion
2. [Setup from binary release](setup/setup-from-binary-release.md)
```

File: site/docs/ch/quickstart/quick-demo.md [hunk 12]
```diff
@@ -0,0 +1,169 @@
...
+
+
+
+# Prepare 1: create catalog
+
+在浏览器打开 [http://localhost:1630](http://localhost:1630) 进入 Dashboard 页面，输入 admin/admin  登录。
+点击侧边栏 Catalogs ，然后点击 Catalog List 下的 `+` 按钮，添加第一个 Catalog， 设置其名字为 `demo_catalog`。
+请按照以下截图设置 Catalog 基本配置
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/quick-demo.md:15]:
```suggestion
请按照以下截图设置 Catalog 基本配置：
```

File: site/docs/ch/quickstart/quick-demo.md [hunk 13]
```diff
@@ -0,0 +1,169 @@
...
+
+![Create catalog](../images/quickstart/create-catalog.png)
+
+注：对于 Hadoop 配置文件，如果是采用 Docker setup，配置文件在 `<ARCIT-WORKSPACE>/hadoop-config` 目录，否则不用上传任何文件。
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/quick-demo.md:19]:
```suggestion
???+note "对于 Hadoop 配置文件，如果是采用 Docker setup，配置文件在 `<ARCIT-WORKSPACE>/hadoop-config` 目录，否则不用上传任何文件。"
```

File: site/docs/ch/quickstart/quick-demo.md [hunk 14]
```diff
@@ -0,0 +1,169 @@
...
+
+注：对于 warehouse.dir，如果是本地部署，填写一个本地目录即可，需要有 Hadoop Username 配置的用户的访问权限。
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/quick-demo.md:21]:
```suggestion
???+note "对于 warehouse，如果是本地部署，填写一个本地目录即可，需要有 Hadoop Username 配置的用户的访问权限。"
```

File: site/docs/ch/quickstart/quick-demo.md [hunk 15]
```diff
@@ -0,0 +1,169 @@
...
+
+# Prepare 2: start optimizers
+
+AMS默认配置下会有一个类型为 local 的 optimizer group，这里需要在此 group 下创建一个 optimizer。
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/quick-demo.md:25]:
```suggestion
AMS 默认配置下会有一个类型为 local 的 optimizer group，这里需要在此 group 下创建一个 optimizer。 
```

File: site/docs/ch/quickstart/quick-demo.md [hunk 16]
```diff
@@ -0,0 +1,169 @@
...
+注：对于 warehouse.dir，如果是本地部署，填写一个本地目录即可，需要有 Hadoop Username 配置的用户的访问权限。
+
+# Prepare 2: start optimizers
+
+AMS默认配置下会有一个类型为 local 的 optimizer group，这里需要在此 group 下创建一个 optimizer。 
+进入 AMS 的 Optimizing 页面，选择 Optimizers。
+
+![Optimizers](../images/quickstart/Optimizing.png)
+
+点击 `Scale-Out` 按钮选择对应 `Optimizer Group` 并且配置 optimizer 并发度，点击OK后即完成了 optimizer 的创建。
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/quick-demo.md:30]:
```suggestion
点击 `Scale-Out` 按钮选择对应 `Optimizer Group` 并且配置 optimizer 并发度，点击 `OK` 后即完成了 optimizer 的创建。
```

File: site/docs/ch/quickstart/quick-demo.md [hunk 17]
```diff
@@ -0,0 +1,169 @@
...
+    'format' = 'changelog-csv',
+    'changelog-csv.column-delimiter' = '|'
+);
+
+-- 关闭 Flink 引擎对 Delete 数据的过滤
+set table.exec.sink.upsert-materialize=none;
+
+-- 往 Arctic 表实时写入数据
+INSERT INTO `arctic`.`db`.`user`
+/*+OPTIONS('flink.max-continuous-empty-commits'='9223372036854775807')*/
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/quick-demo.md:111]:
```suggestion
```

File: site/docs/ch/quickstart/quick-demo.md [hunk 18]
```diff
@@ -0,0 +1,169 @@
...
+DELETE|4|rock|2022-07-02 09:01:00
+DELETE|3|lee|2022-07-01 10:11:00
+
+```
+
+然后通过 Terminal 查询数据，预期数据为：
+
+![Upsert result2](../images/quickstart/upsert-result2.png)
+
+
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/quick-demo.md:158]: Too many empty rows.

File: site/docs/ch/quickstart/setup/setup-from-binary-release.md [hunk 19]
```diff
@@ -0,0 +1,88 @@
...
+
+Arctic 提供了基于 Docker Compose 部署的 Quick Start 环境，
+如果您的环境不方便安装 Docker 以及相关工具，您也可以尝试直接通过以下方式直接部署 Arctic Demo Cluster.
+
+# Requirements
+
+1. 在开始之前，请先确保您安装了 Java 8 并且设置好了 JAVA_HOME 环境变量。
+2. 请确保您的环境变量中没有 HADOOP_HOME，HADOOP_CONF_DIR，如果有请先 unset 这些环境变量
+
+
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/setup/setup-from-binary-release.md:11]: no needed empty row.

File: site/docs/ch/quickstart/quick-demo.md [hunk 20]
```diff
@@ -0,0 +1,169 @@
...
+然后输入测试数据(需要回车以输入最后一行数据)。
+
+```shell
+INSERT|4|rock|2022-07-02 09:01:00
+INSERT|5|jack|2022-07-02 12:11:40
+INSERT|6|mars|2022-07-02 11:19:10
+
+```
+
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/quick-demo.md:132]: no needed empty row.

File: site/docs/ch/quickstart/setup/setup-from-binary-release.md [hunk 21]
```diff
@@ -0,0 +1,88 @@
...
+export ARCTIC_VERSION=0.4.0
+export RELEASE_TAG=v0.4.0-rc1
+# 下载二进制分发包
+wget https://github.com/NetEase/arctic/releases/download/${RELEASE_TAG}/arctic-${ARCTIC_VERSION}-bin.zip
+
+# 解压文件
+unzip arctic-${ARCTIC-VERSION}-bin.zip
+cd arctic-${ARCTIC-VERSION}
+./bin/ams.sh start
+
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/setup/setup-from-binary-release.md:30]: no needed empty row.

File: site/docs/ch/quickstart/setup/setup-from-docker.md [hunk 22]
```diff
@@ -0,0 +1,46 @@
+# Setup From Docker
+
+Arctic 提供了基于 Docker 部署的 Quick Start 环境，基于此部分的文档可以完成后续 Quick Demo 的所有环境准备工作。
+
+
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/setup/setup-from-docker.md:5]: no needed empty row.

File: site/docs/ch/quickstart/setup/setup-from-docker.md [hunk 23]
```diff
@@ -0,0 +1,46 @@
...
+在开始基于 Docker 部署 Arctic 前，请确保您的主机上已经安装了 Docker-Compose 环境，关于 Docker 如何安装请参考 
+[https://docs.docker.com/get-docker/](https://docs.docker.com/get-docker/)
+
+注：建议在 Linux 或 MacOs 进行操作，如果您使用的是 Windows 系统，可以考虑使用 WSL2 ，关于如何开启 WSL2 以及安装 Docker 
+请参考 [https://docs.docker.com/desktop/install/windows-install/](https://docs.docker.com/desktop/install/windows-install/)
+
+在完成 Docker 安装后，请确保安装了 Docker-Compose 工具，关于 Docker-Compose 工具如何安装请参考
+[https://github.com/docker/compose-cli/blob/main/INSTALL.md](https://github.com/docker/compose-cli/blob/main/INSTALL.md)
+
+
```
Comment (zhoujinsong) [inline site/docs/ch/quickstart/setup/setup-from-docker.md:17]: no needed empty row.

**Review by zhoujinsong — COMMENTED** [review]
@baiyangtx  I left some comments.

File: site/docs/ch/quickstart/quick-demo.md (hunk 20 above)
Comment (baiyangtx) [reply site/docs/ch/quickstart/quick-demo.md:132]: Here is needed, if no empty row, the last record will not be read by the flink source.

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**